datasets/gov/sumin/vintra.csv

```diff
@@ -0,0 +1,21 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/sumin/vintra,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,Agency,,,,,,,,,,
+,,,,,name,string,,,,,,,Transporto duomenų teikėjo pavadinimas,
+,,,,,,,,,,,,,,
+,,,,VehicleType,,,,,,,,,,
+,,,,,type,string,,,,,,,Transporto priemonės rūšis,
+,,,,,,,,,,,,,,
+,,,,TransportPosition,,,,,,,,,,
+,,,,,agency,ref,Agency,,,,,,Duomenų teikėjas,
+,,,,,vehicle_type,ref,VehicleType,,,,,,Transporto priemonės rūšis,
+,,,,,route,string,,,,,,,Maršrutas,
+,,,,,track_id,string,,,,,,,Reiso identifikatorius,
+,,,,,vehicle_id,string,,,,,,,Transporto priemonės identifikacinis  numeris,
+,,,,,position,"geometry(point, 4326)",,,,,,,Transporto priemonės pozicija,
+,,,,,speed,integer,,,,,,,Transporto priemonės greitis,
+,,,,,bearing,integer,,,,,,,Azimutas,
+,,,,,trip_start_time,time,,,,,,,Reiso pradžia,
+,,,,,delay,integer,,,,,,,Nuokrypis nuo grafiko sekundėmis,
+,,,,,timestamp,datetime,,,,,,,Pozicijos gavimo laikas,
```

Comment:
Would you consider normalizing data? All open data should be in normalized form, especially data entities with huge amounts of data. So here we would like to minimize amount of data, for example:

```
,,,,Vehicle,,,,,,,,,Transporto priemonė,
,,,,,agency,ref,Agency,,,,,,Auto parkas,
,,,,,id,string,,,,,,,Identifikacinis  numeris,
,,,,,type,ref,VehicleType,,,,,,Transporto priemonės rūšis,
,,,,,,,,,,,,,,
,,,,Route,,,,,,,,,Maršrutas,
,,,,,name,string,,,,,,,Pavadinimas,
,,,,,,,,,,,,,,
,,,,Track,,,,,,,,,Reisas,
,,,,,id,string,,,,,,,Identifikatorius,
,,,,,route,string,,,,,,,Maršrutas,
,,,,,,,,,,,,,,
,,,,Trip,,,,,,,,,Kelionė,
,,,,,track,ref,Track,,,,,,Reisas,
,,,,,vehicle,ref,Vehicle,,,,,,Transporto priemonė,
,,,,,start_time,time,,,,,,,Reiso pradžia,
,,,,,,,,,,,,,,
,,,,Position,,,,,,,,,Transporto priemonės laikas ir vieta,
,,,,,trip,ref,Trip,,,,,,Reisas,
,,,,,position,"geometry(point, 4326)",,,,,,,Pozicija,
,,,,,speed,integer,km/h,,,,,,Greitis,
,,,,,bearing,integer,°,,,,,,Azimutas,
,,,,,delay,integer,s,,,,,,Nuokrypis nuo grafiko,
,,,,,timestamp,datetime,,,,,,,Pozicijos gavimo laikas, 
```

Also consider removing `speed`, `bearing` and `delay` since these are derived values. Data user will be able to calculated those values from given data. In order to calculate `delay` a full time table data should also must be made available.

By reduce the biggest table to just three columns `trip`, `position` and `timestamp` we will be able to fit a lot more data.